File: Even Divison/Code.java
import java.util.*;
import java.io.*;

class Main {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        long arr[]=new long[n];
        for(int i=0;i<n;i++){
            arr[i]=sc.nextLong();
        }
        for(int i=0;i<n;i++){
            if(arr[i]==2){
                System.out.println("NO");
            }
            else if(arr[i]%2==0){
                System.out.println("YES");
            }
            else{
                System.out.println("NO");
            }
        }
    }
}

File: Palindrome Case Insensitive/Code.js
import java.util.*;
import java.io.*;

class Main {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        // sc.next();
        for(int i=0;i<t;i++){
            String str=sc.next();
            str = str.toLowerCase();
            StringBuffer sb=new StringBuffer(str);
            sb.reverse();

            if(str.contentEquals(sb)){
                System.out.println("Yes");
            }
            else{
                System.out.println("No");
            }
        }
        
        
        
        
        
    }
}
